c/unixFileService.c

```diff
@@ -981,6 +981,43 @@ static int serveUnixFileChangeOwner (HttpService *service, HttpResponse *respons
   return 0;
 }
 
+static int serveUnixFileChangeTag (HttpService *service, HttpResponse *response) {
+  HttpRequest *request = response->request;
+  char *routeFileFrag = stringListPrint(request->parsedFile, 2, 1000, "/", 0);
+
+  if (routeFileFrag == NULL) {
+   respondWithError(response, HTTP_STATUS_BAD_REQUEST, "File name not provided");
+   return 0;
+  }
+
+  char *encodedRouteFileName = stringConcatenate(response->slh, "/", routeFileFrag);
+  char *routeFileName = cleanURLParamValue(response->slh, encodedRouteFileName);
+
+  char *codepage  = getQueryParam(response->request, "codeset");
+  char *recursive = getQueryParam(response->request, "recursive");
+  char *pattern   = getQueryParam(response->request, "pattern");
+  char *type      = getQueryParam(response->request, "type");
+
+  if (codepage == NULL || recursive == NULL || type == NULL) {
```

Comment:
@ifakhrutdinov  I removed the null check for pattern because it is not a required parameter. And already there is a null check on the path

Reply:
aslo codepage if it is binary in type we can't define a code no it seems.

Reply:
Do we want to make it required when it's not binary? What do similar services do?

Reply:
Yes we can make it required when the type is not binary

Reply:
its already been covered @ifakhrutdinov 

![Screenshot from 2020-08-03 12-37-57](https://user-images.githubusercontent.com/20354461/89213627-a9aa3800-d5c5-11ea-84f0-25586c38f708.png)